lib/expander/src/Events/CourseModuleCompleted.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nikolay.mikov
+ * Date: 5.6.2017 г.
+ * Time: 11:32
+ */
+
+namespace LogExpander\Events;
+
+
+class CourseModuleCompleted extends  Event
+{
+    public function read(array $opts)
+    {
+
+        global $DB;
```

Comment:
By the way  I want to use  the readStoreRecord  method  but  I don't inherit it's class. The function is with access modifier "protected"  only the classes which inherit Repo class  can use that function  What do you mean by  using  $this->repo ?